lib/formatter-webpack.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const StylelintPluginError = require('./StylelintPluginError');
```

Comment:
another minor nit, the filename should be kebab-cased version of the export, i.e. `stylelint-plugin-error`.

Reply:
ah you're requesting that change in the classes file. that's pretty unusual for a file that exports a class :)